scripts/installGit.ps1

```diff
@@ -11,72 +11,76 @@ param (
     [string]
     $Branch,
 
+    [Parameter(Mandatory=$true)]
+    [string]
+    $GitUserName,
+
     [boolean]
     $CalledUsingRemotePS = $false,
 
     [boolean]
     $InstallGit = $true
     )
 
-Write-Debug "GitRepo = $GitRepo" | Out-Host
-Write-Debug "GitRepoPath = $GitRepoPath" | Out-Host
+Write-Debug "GitRepo = $GitRepo" | Out-Default | Write-Host
```

Comment:
Write-xxxxx cmdlets must not use Out-Default. Changing to Write-Host is correct though. so should be
```
| Write-Host
```